src/index.ts

```diff
@@ -27,6 +31,7 @@ app.use(
   )
 );
 app.use(cors({ maxAge: 86400 }));
+app.use(favicon(path.join(__dirname, '../public', 'favicon.ico')));
```

Comment:
can we use this approach without any extra libraries?
```suggestion
app.get('/favicon.ico', (req, res) => res.status(204)); 
```

Reply:
What you're suggesting will not serve favicon at all

Reply:
yes, why do you need it for API?

Reply:
There's the `/preview` endpoint, which is not an API.

And without this, polling the `/metrics` endpoint will log the favicon, and increase counters

Reply:
To test: 
- comment the favicon line in index.ts
- visit /metrics, and hard refresh the page
- you will see a counter for the favicon request
- refresh the page
- the favicon counter increase
- => this is undesired, as each call to the /metric endpoint will increase the GET counter

Now, without commenting the favicon line, doing the same steps above, there is no favicon counters in the metrics page